apps/web/src/features/coaches/components/coaches-table-columns.tsx

```
@@ -87,33 +71,32 @@ export const coachesTableColumns = [
-	columnHelper.accessor((row) => row.team?.premier_coach?.name, {
-		id: "premier_coach",
-		header: "Assigned Premier Coach",
-		enableColumnFilter: true,
-		enableSorting: true,
-		cell: ({ row }) => {
-			const premierCoachName = row.original.team?.premier_coach?.name;
+      return (
+        <div className="flex flex-wrap gap-1">
+          {rolesList.map((role, index) => (
+            <span
+              key={index}
```

Biome (2.1.2)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)